test/js-api/jsapi.js

```diff
@@ -714,7 +714,22 @@ test(() => {
     assert_equals(instantiate, instantiateDesc.value);
     assert_equals(instantiate.length, 1);
     assert_equals(instantiate.name, "instantiate");
-    function assertInstantiateError(args, err) {
+
+    // Keep track of all the messages for assertInstantiateError and assertCompileSuccess
+    // to make sure they're all unique.
+    const testNames = new Set();
+    function ensureUniqueTest(message) {
+        if (testNames.has(message)) {
+            if (quit instanceof Function) {
```

Comment:
Done. I wasn't sure what the idiomatic way to check whether I should or can call something is.